packages/ember-testing/lib/events.js

```diff
@@ -38,11 +38,16 @@ export function fireEvent(element, type, options = {}) {
     let x = rect.left + 1;
     let y = rect.top + 1;
     let simulatedCoordinates = {
-      screenX: x + 5,
-      screenY: y + 95,
-      clientX: x,
-      clientY: y
+      screenX: options.screenX || x + 5,
```

Comment:
what if the option provided is the coordinate of `0` it will be falsey, and the `||` will fall to the right hand side, is that intentional?